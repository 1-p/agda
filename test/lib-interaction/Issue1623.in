top_command (cmd_load currentFile [".","../../agda-stdlib/src/"])
goal_command 0 cmd_auto ""
