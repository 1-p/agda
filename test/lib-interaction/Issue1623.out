Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue1623 (Issue1623.agda). " t)
(agda2-info-action "*Type-checking*" " Skipping Level (/home/abel/agda/agda-stdlib/src/Level.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Function (/home/abel/agda/agda-stdlib/src/Function.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Empty (/home/abel/agda/agda-stdlib/src/Data/Empty.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Nullary (/home/abel/agda/agda-stdlib/src/Relation/Nullary.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Product (/home/abel/agda/agda-stdlib/src/Data/Product.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.Core (/home/abel/agda/agda-stdlib/src/Relation/Binary/Core.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.Consequences.Core (/home/abel/agda/agda-stdlib/src/Relation/Binary/Consequences/Core.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.PropositionalEquality.Core (/home/abel/agda/agda-stdlib/src/Relation/Binary/PropositionalEquality/Core.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Unit.Base (/home/abel/agda/agda-stdlib/src/Data/Unit/Base.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Bool.Base (/home/abel/agda/agda-stdlib/src/Data/Bool/Base.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Maybe.Base (/home/abel/agda/agda-stdlib/src/Data/Maybe/Base.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Sum (/home/abel/agda/agda-stdlib/src/Data/Sum.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.Consequences (/home/abel/agda/agda-stdlib/src/Relation/Binary/Consequences.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.Indexed.Core (/home/abel/agda/agda-stdlib/src/Relation/Binary/Indexed/Core.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary (/home/abel/agda/agda-stdlib/src/Relation/Binary.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Nat.Base (/home/abel/agda/agda-stdlib/src/Data/Nat/Base.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.List.Base (/home/abel/agda/agda-stdlib/src/Data/List/Base.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.DifferenceList (/home/abel/agda/agda-stdlib/src/Data/DifferenceList.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.Indexed (/home/abel/agda/agda-stdlib/src/Relation/Binary/Indexed.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.HeterogeneousEquality.Core (/home/abel/agda/agda-stdlib/src/Relation/Binary/HeterogeneousEquality/Core.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.PreorderReasoning (/home/abel/agda/agda-stdlib/src/Relation/Binary/PreorderReasoning.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.EqReasoning (/home/abel/agda/agda-stdlib/src/Relation/Binary/EqReasoning.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Unit.NonEta (/home/abel/agda/agda-stdlib/src/Data/Unit/NonEta.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Function.Equality (/home/abel/agda/agda-stdlib/src/Function/Equality.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.PropositionalEquality (/home/abel/agda/agda-stdlib/src/Relation/Binary/PropositionalEquality.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Unit (/home/abel/agda/agda-stdlib/src/Data/Unit.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.AVL (/home/abel/agda/agda-stdlib/src/Data/AVL.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Category.Functor (/home/abel/agda/agda-stdlib/src/Category/Functor.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Category.Applicative.Indexed (/home/abel/agda/agda-stdlib/src/Category/Applicative/Indexed.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Category.Applicative (/home/abel/agda/agda-stdlib/src/Category/Applicative.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Function.Injection (/home/abel/agda/agda-stdlib/src/Function/Injection.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Function.Equivalence (/home/abel/agda/agda-stdlib/src/Function/Equivalence.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Nullary.Decidable (/home/abel/agda/agda-stdlib/src/Relation/Nullary/Decidable.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.PartialOrderReasoning (/home/abel/agda/agda-stdlib/src/Relation/Binary/PartialOrderReasoning.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Nat (/home/abel/agda/agda-stdlib/src/Data/Nat.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Fin (/home/abel/agda/agda-stdlib/src/Data/Fin.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Vec (/home/abel/agda/agda-stdlib/src/Data/Vec.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Unary (/home/abel/agda/agda-stdlib/src/Relation/Unary.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Function.LeftInverse (/home/abel/agda/agda-stdlib/src/Function/LeftInverse.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Function.Surjection (/home/abel/agda/agda-stdlib/src/Function/Surjection.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Function.Bijection (/home/abel/agda/agda-stdlib/src/Function/Bijection.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Function.Inverse (/home/abel/agda/agda-stdlib/src/Function/Inverse.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.List.Pointwise (/home/abel/agda/agda-stdlib/src/Relation/Binary/List/Pointwise.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.List.StrictLex (/home/abel/agda/agda-stdlib/src/Relation/Binary/List/StrictLex.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.On (/home/abel/agda/agda-stdlib/src/Relation/Binary/On.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Char.Core (/home/abel/agda/agda-stdlib/src/Data/Char/Core.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.PropositionalEquality.TrustMe (/home/abel/agda/agda-stdlib/src/Relation/Binary/PropositionalEquality/TrustMe.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.String.Base (/home/abel/agda/agda-stdlib/src/Data/String/Base.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Char.Base (/home/abel/agda/agda-stdlib/src/Data/Char/Base.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.FunctionProperties.Core (/home/abel/agda/agda-stdlib/src/Algebra/FunctionProperties/Core.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.FunctionProperties (/home/abel/agda/agda-stdlib/src/Algebra/FunctionProperties.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.Structures (/home/abel/agda/agda-stdlib/src/Algebra/Structures.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Vec.N-ary (/home/abel/agda/agda-stdlib/src/Data/Vec/N-ary.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.Reflection (/home/abel/agda/agda-stdlib/src/Relation/Binary/Reflection.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra (/home/abel/agda/agda-stdlib/src/Algebra.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.Operations (/home/abel/agda/agda-stdlib/src/Algebra/Operations.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.Properties.Group (/home/abel/agda/agda-stdlib/src/Algebra/Properties/Group.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.Properties.AbelianGroup (/home/abel/agda/agda-stdlib/src/Algebra/Properties/AbelianGroup.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.Properties.Ring (/home/abel/agda/agda-stdlib/src/Algebra/Properties/Ring.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.Morphism (/home/abel/agda/agda-stdlib/src/Algebra/Morphism.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.RingSolver.AlmostCommutativeRing (/home/abel/agda/agda-stdlib/src/Algebra/RingSolver/AlmostCommutativeRing.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.RingSolver.Lemmas (/home/abel/agda/agda-stdlib/src/Algebra/RingSolver/Lemmas.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.RingSolver (/home/abel/agda/agda-stdlib/src/Algebra/RingSolver.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.RingSolver.Simple (/home/abel/agda/agda-stdlib/src/Algebra/RingSolver/Simple.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Nat.Properties.Simple (/home/abel/agda/agda-stdlib/src/Data/Nat/Properties/Simple.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Nat.Properties (/home/abel/agda/agda-stdlib/src/Data/Nat/Properties.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Char (/home/abel/agda/agda-stdlib/src/Data/Char.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Coinduction (/home/abel/agda/agda-stdlib/src/Coinduction.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Category.Monad.Indexed (/home/abel/agda/agda-stdlib/src/Category/Monad/Indexed.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Category.Monad (/home/abel/agda/agda-stdlib/src/Category/Monad.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.List (/home/abel/agda/agda-stdlib/src/Data/List.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Bool (/home/abel/agda/agda-stdlib/src/Data/Bool.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.Properties.Lattice (/home/abel/agda/agda-stdlib/src/Algebra/Properties/Lattice.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.Properties.DistributiveLattice (/home/abel/agda/agda-stdlib/src/Algebra/Properties/DistributiveLattice.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.Properties.BooleanAlgebra (/home/abel/agda/agda-stdlib/src/Algebra/Properties/BooleanAlgebra.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Bool.Properties (/home/abel/agda/agda-stdlib/src/Data/Bool/Properties.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.List.NonEmpty (/home/abel/agda/agda-stdlib/src/Data/List/NonEmpty.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.BoundedVec.Inefficient (/home/abel/agda/agda-stdlib/src/Data/BoundedVec/Inefficient.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Plus (/home/abel/agda/agda-stdlib/src/Data/Plus.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Fin.Properties (/home/abel/agda/agda-stdlib/src/Data/Fin/Properties.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Vec.Equality (/home/abel/agda/agda-stdlib/src/Data/Vec/Equality.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.InducedPreorders (/home/abel/agda/agda-stdlib/src/Relation/Binary/InducedPreorders.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Function.Related (/home/abel/agda/agda-stdlib/src/Function/Related.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.List.Any (/home/abel/agda/agda-stdlib/src/Data/List/Any.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Category.Monad.Identity (/home/abel/agda/agda-stdlib/src/Category/Monad/Identity.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.HeterogeneousEquality (/home/abel/agda/agda-stdlib/src/Relation/Binary/HeterogeneousEquality.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Vec.Properties (/home/abel/agda/agda-stdlib/src/Data/Vec/Properties.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Binary.Vec.Pointwise (/home/abel/agda/agda-stdlib/src/Relation/Binary/Vec/Pointwise.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Algebra.Properties.BooleanAlgebra.Expression (/home/abel/agda/agda-stdlib/src/Algebra/Properties/BooleanAlgebra/Expression.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Fin.Subset (/home/abel/agda/agda-stdlib/src/Data/Fin/Subset.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Fin.Subset.Properties (/home/abel/agda/agda-stdlib/src/Data/Fin/Subset/Properties.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Fin.Dec (/home/abel/agda/agda-stdlib/src/Data/Fin/Dec.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Relation.Nullary.Negation (/home/abel/agda/agda-stdlib/src/Relation/Nullary/Negation.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Conat (/home/abel/agda/agda-stdlib/src/Data/Conat.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.Colist (/home/abel/agda/agda-stdlib/src/Data/Colist.agdai). " t)
(agda2-info-action "*Type-checking*" " Skipping Data.String (/home/abel/agda/agda-stdlib/src/Data/String.agdai). " t)
(agda2-info-action "*Type-checking*" "Finished Issue1623. " t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Relation.Binary.IsStrictTotalOrder .Relation.Binary.Core.Dummy._≡_ (Relation.Binary.StrictTotalOrder._<_ strictTotalOrder) " nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> (agda2-status-action "")
(agda2-info-action "*Auto*" "" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> 